src/emulator/storage/apis/gcloud.ts

```diff
@@ -219,6 +209,11 @@ export function createCloudEndpoints(emulator: StorageEmulator): Router {
       if (emulatorInfo === undefined) {
         return res.sendStatus(500);
       }
+      const name = getValidName(req.query, req.body);
+      if (name == null) {
```

Comment:
go/js-style#features-equality-checks, please use ===

Reply:
The goal is to catch null and undefined with this check I think its an exception to the rule, go/js-style#features-equality-checks-exceptions

Reply:
See comments below, since `getValidName` returns `string | undefined`, you should only need `name === undefined` here since `getValidName` can't return null

Reply:
yeah thats fair, I'll swap it to name === undefined